### PR TITLE
[IMP] Add Swisstopo WMTS layers support

### PR DESCRIPTION
**Pull request by asaunier**
screenshot from 2019-02-28 19-18-09

screenshot from 2019-02-28 19-15-40

The same changes are also available in v11, see https://github.com/asaunier/geospatial/commit/048a6bd90fafbedfca2fe06e23e18fab36492840

### CODE REVIEW

**Comment by asaunier** [comment]
@yvaucher Looks good to you?

File: base_geoengine/static/src/js/geoengine_common.js [hunk 1]
```diff
@@ -121,6 +121,34 @@ odoo.define('base_geoengine.geoengine_common', function (require) {
...
+                                url = url.replace('{i}', i);
+                                url = url.replace('{layer}', l.layername || 'ch.swisstopo.pixelkarte-farbe');
+                                url = url.replace('{time}', l.time || 'current');
+                                url = url.replace('{format}', l.format_suffix || 'jpeg');
+                                urls.push(url);
+                            });
+                            var source = new ol.source.XYZ({
+                                attributions: [
+                                    new ol.Attribution({
+                                        html: '<a target="_blank" href="http://www.swisstopo.admin.ch">swisstopo</a>'
```
Comment (fredj) [inline base_geoengine/static/src/js/geoengine_common.js]:
```suggestion
                                        html: '<a target="_blank" href="https://www.swisstopo.admin.ch">swisstopo</a>'
```

**Comment by asaunier** [comment]
@fredj Thanks for your change suggestion! Committed :)

**Comment by yvaucher** [comment]
@asaunier the change is smaller than I would have thought. Nevertheless, would it be possible to push that in a separate module? I don't think it make much sense to have it there.

**Comment by asaunier** [comment]
@yvaucher Makes sense! I'll do this and close the current PR.
Would make things easier for me if your PR https://github.com/OCA/geospatial/pull/185 is merged before, any chance it is merged soon :P

**Comment by yvaucher** [comment]
Closing in favor of OCA#187

**Pull request closed without merging**